src/client/index.js

```diff
@@ -1,6 +1,10 @@
 'use strict';
 
 const ReactDOM = require('react-dom');
-const React = require('react'); // eslint-disable-line no-unused-vars
+const React = require('react');
+import SignupComponent from './signupComponent.js';
```

Comment:
- we should use a folder for the components
 - Component files are always Capitalized